packages/slate/src/models/node.js

```diff
@@ -1976,6 +1976,25 @@ class Node {
   validate(schema) {
     return schema.validateNode(this)
   }
+
+  /**
+   * The first descendant key requiring validation
+   *
+   * @param {Schema} schema
+   * @return {String|Null}
+   */
+
+  getFirstInvalidDescendantKey(schema) {
```

Comment:
Can we change this to not return the `key` but return the node itself? Since the key can always be accessed by just doing `node.key` at that point.

And then change the name to `getFirstInvalidDescendant` instead.